sandbox/katas/Greed.01/Greed/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualBasic;
+
+var greedGame = new GreedGame();
+
+var dices = greedGame.ReturnDices();
+
+foreach (var d in dices)
+{
+    Console.WriteLine(d);
+}
+
+var results = greedGame.CountNumbers(dices);
```

Comment:
toto by som možno volal z metódy CountScore. Je to medzivýpočet, ktorý by de facto Program.cs nemal vôbec zaujímať :-)